### ISSUE TITLE
Environment Error

### ISSUE THREAD

**Issue by shuowang666** [exitudio/MMM#5]
<img width="806" alt="截屏2024-04-09 20 11 42" src="https://github.com/exitudio/MMM/assets/56186807/35ac3651-e1f2-4682-a56c-625fd7d70afe">
When creating the environment, I encountered this error, how to solve it?

**Comment by exitudio** [comment]
Hi,
Since we developed based on T2M-GPT, you can try to install env from [T2M-GPT env](https://github.com/Mael-zys/T2M-GPT?tab=readme-ov-file#21-environment).
Or install packages manually.

**Comment by shuowang666** [comment]
Thanks for your answer！
Do I need to adjust the learning rate or other parameters during multi-gpu training?

**Comment by exitudio** [comment]
Using multiple gpus shouldn't effect the training. 
We increase batch size proportional to the decreasing number of iterations to make it train faster.  For example:
```
MULTI_BATCH=4
...
--batch-size $((128*MULTI_BATCH)) \
--total-iter $((300000/MULTI_BATCH)) \
--lr-scheduler $((150000/MULTI_BATCH)) \
--eval-iter $((20000/MULTI_BATCH)) \
```

However, with bigger batch size, we got similar results regarding different batch sizes with the same learning rate of "0.0001".

**Comment by shuowang666** [comment]
Thanks very much!

**Comment by exitudio** [comment]
I just retrained everything again. Seem like only 128 batch size works better along with ```--num-local-layer 2```. I got FID ~.070. 
I already updated the pretrain model with log.
